app/views/return-requirements/check.njk

```diff
@@ -309,12 +314,16 @@
             }
           ]
         }) }}
+        {% else %}
+          <h3 class="govuk-heading-m">Returns are not required for this licence</h3>
+          <hr class="govuk-!-margin-bottom-5 govuk-section-break--s govuk-section-break--visible">
       {% endif %}
 
       <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
 
       {{ govukButton({
         html: '<span data-test="meta-data-approve">' + "Approve returns requirements" + "</span>",
+        text: "Approve returns requirements",
```

Comment:
I could have sworn I flagged this on the previous PR!

Anyway, according to [the docs](https://design-system.service.gov.uk/components/button/) if the `html:` option is provided `text:` will be ignored. However, in this case we don't need to provide a `html:` option. Someone has done this to give it a Cypress test attribute, but we don't need these for buttons.

I'd delete the `html:` option and keep the `text:` you added.

Reply:
I was wondering why this was an issue. I must have messed up the initial branch.